apps/generator/lib/filtersRegistry.js

```diff
@@ -21,6 +22,7 @@ module.exports.registerFilters = async (nunjucks, templateConfig, templateDir, f
 
 /**
  * Registers the local template filters.
+ * @deprecated since version 3.0
```

Comment:
i have assumed this to be included in the next major release, is this fine @derberg ?